test/utils/testconfig/testconfig.go

```diff
@@ -225,39 +284,184 @@ func GetDesiredConfig(forceUpdate bool) TestConfig {
 
 // Create ptpconfigs
 func CreatePtpConfigurations() {
-	var err error
 	// Initialize desired ptp config for all configs
 	GetDesiredConfig(true)
 	// in multi node configuration create ptp configs
 	if GlobalConfig.PtpModeDesired != Discovery {
-		GlobalConfig.L2Config, err = l2discovery.GetL2DiscoveryConfig()
+		// Initialize l2 library
+		l2lib.GlobalL2DiscoveryConfig.SetL2Client(client.Client, client.Client.Config)
+
+		// Collect L2 info
+		config, err := l2lib.GlobalL2DiscoveryConfig.GetL2DiscoveryConfig()
```

Comment:
Need to add additional ptp capable filter in l2 discovery (was done here before) - in progress

Reply:
fixed

Reply:
fixed